tests/testthat/test-get_source_expressions.R

```diff
@@ -170,20 +170,20 @@ test_that("Can extract line number from parser errors", {
 })
 
 test_that("1- or 2-width octal expressions give the right STR_CONST values", {
-  with_content_to_parse("'\\1'", expect_identical(pc[[1L]][2L, "text"], "'\\1'"))
-  with_content_to_parse('"\\1"', expect_identical(pc[[1L]][2L, "text"], '"\\1"'))
+  with_content_to_parse("'\\1'", expect_identical(pc[[1L]][1L, "text"], "'\\1'"))
```

Comment:
probably it would be better to test this directly against `getParseData()` somehow... someday...